segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeperstorage/LedgerData.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.bookkeeperstorage;
+
+import java.nio.ByteBuffer;
+import javax.annotation.concurrent.GuardedBy;
+import lombok.Data;
+import org.apache.bookkeeper.client.LedgerHandle;
+
+/**
+ * Stores metadata about a single BookKeeper Ledger.
+ */
+@Data
+class LedgerData {
+    // Retrieved from ZK
```

Comment:
This will be used as Javadoc (because of the `@Data` tag). Please use Javadoc comment styles.